### PR TITLE
[kube-prometheus-stack] Delete statefulset Grafana dashboard

### PR DESCRIPTION
**Pull request by tamalsaha**
Signed-off-by: Tamal Saha <tamal@appscode.com>

#### What this PR does / why we need it:
This was removed from the mixin project. But it was not removed from the chart.
https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/598

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
please update chart version

**Comment by tamalsaha** [comment]
done!

**Comment by monotek** [comment]
Have you checked the update script for needed changes?
https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py

**Comment by tamalsaha** [comment]
Yes. I deleted the templates in this folder and then generated using that script.

**Review by monotek — DISMISSED** [review]

**Comment by tamalsaha** [comment]
ping!

**Review by monotek — APPROVED** [review]

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -23,8 +23,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 34.10.1
-appVersion: 0.55.0
+version: 35.1.0
+appVersion: 0.57.0
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:27]:
```suggestion
appVersion: 0.56.0
```

File: charts/kube-prometheus-stack/Chart.yaml [hunk 2]
```diff
@@ -23,8 +23,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 35.0.0
-appVersion: 0.56.0
+version: 35.1.0
```
Comment (monotek) [inline charts/kube-prometheus-stack/Chart.yaml:26]:
```suggestion
version: 35.0.1
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Comment by tamalsaha** [comment]
Done!

**Review by monotek — APPROVED** [review]

**Comment by tamalsaha** [comment]
Thanks!

**Pull request merged**